test/constants.js

```diff
@@ -1,9 +1,46 @@
 import path from 'path';
+import {
+  AUTHENTICATED,
+  DEFAULT_SYNC_MODE,
+  DEFAULT_USER_MODE,
+} from '../src/config/constants';
+import { USER_GRAASP } from './fixtures/users';
 
 export const EXPORT_FILEPATH = path.join(__dirname, './tmp/exportSpace');
 export const APPS_FOLDER = 'apps';
 
+export const buildSignedUserForDatabase = ({
```

Comment:
Maybe this function should go in `utils`? Also, by `Signed`, I guess you mean `SignedInUser`. Maybe change the naming.